packages/dataprovider/src/buildWhere.ts

```diff
@@ -413,6 +421,22 @@ const getFilters = (
       (t) => t.name === fieldType.name,
     ) as IntrospectionInputObjectType;
 
+    if (
+      aorFetchType === GET_MANY_REFERENCE &&
+      options.queryDialect === "typegraphql" &&
+      fieldType.name.includes("RelationFilter")
+    ) {
+      return {
+        [originalKey]: {
+          is: {
+            id: {
+              equals: value,
+            },
+          },
+        },
+      };
```

Comment:
shouln't GET_MANY_REFERENCE get many? because this only works if value is not an array